openslides/locale/tinymce/cs.js

```diff
@@ -0,0 +1,219 @@
+tinymce.addI18n('cs',{
```

Comment:
Can you help me to create such a tinymce-i18n repo which will provice i18n files via bower?